plugins/modules/meraki_ms_access_policies.py

```diff
@@ -461,6 +468,12 @@ def main():
             "systems_management_enrollment"
         ],
     }
+    # Execute check for argument completeness
+    if meraki.params["access_policy_type"] == "Hybrid authentication":
+        if isinstance(meraki.params["increase_access_speed"], bool):
```

Comment:
The default type on line 351 should catch this. Is there a corner case which merits the `isinstance()` call?

Reply:
So, this was set when default value was not forced on me by sanity test, I will check and revert you on these changes